Makefile

```diff
@@ -129,14 +129,14 @@ fullinstall2: clean alldeps2 sync_open_data build install
 	mkdir -p src/interface_py/dist2/ && cp -a src/interface_py/dist/*.whl src/interface_py/dist2/
 
 runtime:
-	$(MAKE) py
+	$(MAKE) fullinstall
```

Comment:
Why a fullinstall?

Reply:
We talked about this together before.  The local system won't have the correct C++ backend, xgboost, etc. unless one does make fullinstall.  E.g., if do only make py, then if fullinstall previously was for cuda8, then make py will not be enough to get the cuda9 binary.